roles/common/files/uug_ansible_wrapper.py

```diff
@@ -646,6 +652,42 @@ def unable_to_detect_branch():
         USER_CONFIG['git_branch'] = "master"
         logging.info("Release set to master")
 
+def warn_wrong_release_branch(parent):
+    """
+    Displays a dialog to warn the user that their current branch in the configuration tool does not
+    match their os rlease version.
+    """
+
+    logging.info("The chosen branch is < OS releae. Warning user about switching branches")
+    warning_prompt = (
+        "You are using a version of the configuration tool meant for a different release of Linux Mint. " 
+        "You should consider upgrading braches. The correct configuration settings for your release version of Linux Mint is:"
```

Comment:
Minor wording suggestions
```suggestion
        "You should consider upgrading branches. The recommended configuration settings for your release version of Linux Mint are:"
```

Reply:
Can we make sure this wraps at approx 78-80 chars?